src/themes/elife/lib/dataProvider.ts

```diff
@@ -32,11 +25,9 @@ const getPdfUrl: PdfUrlGetter = async (
       )}.`
     )
   }
-  const response = await query(id, window.fetch)
-  if (pdfType === 'figures') {
-    return Promise.resolve(response.articleData.figuresPdf)
-  }
-  return Promise.resolve(response.articleData.pdf)
+  return (
+    (pdfType === 'figures' ? article.figuresPdf ?? null : article.pdf) ?? ''
```

Comment:
So just comparing this to how the code previously functioned, and seems this changes the behaviour a little. Is this intentional/required? 
Just wondering if `return (pdfType === 'figures' ? article.figuresPdf : article.pdf)` would be more aligned to origin behaviour?

Reply:
spoke to you about this on slack. It's a minor adjustment that I'm accommodating in this PR. We should have been treating the figuresPdf as optional. Could be pulled into a separate PR but we agreed it's okish in this PR.